backend/api/views/compute_plan_graph.py

```diff
@@ -61,8 +61,8 @@ def get_cp_graph(request, compute_plan_pk):
 
     tasks = (
         ComputeTask.objects.filter(compute_plan__key=compute_plan_pk, channel=get_channel_name(request))
-        .annotate(inputs_specs=Subquery(inputs), outputs_specs=Subquery(outputs))
-        .values("key", "rank", "worker", "status", "inputs_specs", "outputs_specs")
+        .annotate(inputs_specs=Subquery(inputs), outputs_specs=Subquery(outputs), function_name=F("function__name"))
+        .values("key", "rank", "function_name", "worker", "status", "inputs_specs", "outputs_specs")
```

Comment:
I'm not sure if this is not missing a `prefetch_related` or a `select_related`, could you add a [n + 1 queries test](https://github.com/Substra/substra-backend/blob/main/backend/api/tests/views/test_views_computetask.py#L831) to check that it's fine? 
We can pair on it if you want.

Reply:
Okay, so the prefetch/select_related is not needed because the `annotate` uses a JOIN underneath https://docs.djangoproject.com/en/4.2/topics/db/aggregation/#joins-and-aggregates

I think it would be worth it adding a comment to mention that fact, because it wasn't obvious (at least for me).